README.md

```diff
@@ -14,17 +14,17 @@ This project is part of ASP.NET 5.0. You can find samples, documentation and get
 ## Run on WebListener/Kestrel:
 * Open a command prompt and cd `\src\MusicStore\`.
 * **[WebListener]:**
-	4. Run `k web` (Application started at URL **http://localhost:5002/**).
+	4. Run `dnx . web` (Application started at URL **http://localhost:5002/**).
```

Comment:
@Praburaj we should use dnx . web and dnx .test to run and test the current application